### PR TITLE
plugins: new export_spec_3d_csv

### PR DESCRIPTION
**Pull request by pieleric**
Allows to export to a CSV a (3D) spectrum data.

As requested by some users.

### CODE REVIEW

File: plugins/export_spec_3d_csv.py [hunk 1]
```diff
@@ -0,0 +1,125 @@
...
+            # Y
+            for iy in range(da.shape[-2]):
+                # X
+                for ix in range(da.shape[-1]):
+                    # Wavelength
+                    for wl, data in zip(spectrum_range, da[:, iy, ix]):
+                        csv_writer.writerow([wl, ix * pxs[0], iy * pxs[1], data])
+                        l += 1
+
+        logging.debug("Exported %d rows", l)
```
Comment (tepals) [inline plugins/export_spec_3d_csv.py:124]: Isn't `l` equal to `da.shape[-2] * da.shape[-1] * len(spectrum_range)`? It looks strange that you have to do the `l+=1` to get the number of rows exported.

File: plugins/export_spec_3d_csv.py [hunk 2]
```diff
@@ -0,0 +1,125 @@
+# -*- coding: utf-8 -*-
+'''
```
Comment (tepals) [inline plugins/export_spec_3d_csv.py:2]: According to PEP these should actually also be triple double quotes. https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring

It looks like most plugins in odemis have triple single quotes, so you can also leave it like this.

File: plugins/export_spec_3d_csv.py (hunk 1 above)
Comment (pieleric) [reply plugins/export_spec_3d_csv.py:124]: Indeed, it *should* be prod(da.shape). During development I wanted to check that it's indeed the case. 
This code is not needed anymore, so I'll remove it.

File: plugins/export_spec_3d_csv.py (hunk 2 above)
Comment (pieleric) [reply plugins/export_spec_3d_csv.py:2]: I agree totally. Actually I think that strings should just always use the double quotes. For consistency I'd like to have no string with single quote. This actually comes from a copy-paste of another plugin. I'll fix this one at least....

**Pull request merged**